pkg/machine/fedorawsl.go

```diff
@@ -0,0 +1,65 @@
+//go:build amd64 || arm64
+// +build amd64 arm64
+
+package machine
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	githubLatestReleaseURL = "https://github.com/gbraad/podman-wsl-fedora/releases/latest/download/rootfs.tar.xz"
+)
+
+func NewFedoraWSLDownloader(vmType, vmName, releaseStream string) (DistributionDownload, error) {
```

Comment:
I think this version should just replace the previous fedora impl, it's only used in the WSL case, and we are unlikely to use it again and could always resurrect from history

Reply:
dropped `fedorawsl.go`